.ruby-version

```diff
@@ -1 +1 @@
-2.5.1
+2.7.6
```

Comment:
Avoids this warning
<img width="971" alt="Screen Shot 2022-07-12 at 1 39 10 PM" src="https://user-images.githubusercontent.com/1791049/178590933-4d6ad543-787a-4e1e-85dc-5d786c211412.png">
.